integration/apps/rspec/README.md

```diff
@@ -78,7 +78,7 @@ e.g. `DD_DEMO_ENV_FEATURES=tracing,profiling`
 
 ### Running integration tests
 
-You can run integration tests using the following and substituting for the Ruby major and minor version (e.g. `2.7`)
+You can run integration tests using the following and substituting for the Ruby major and minor version (e.g. `2.7`). Currently, this integration test has nothing to do in CI, and there are no RSpec tests in `/bin/test`.
```

Comment:
This one is a great find, these tests are leftovers of ddtrace => datadog-ci migration, we need to remove this folder completely. I will do it in a separate PR.